pkg/placement/raft/state.go

```diff
@@ -39,33 +46,46 @@ type DaprHostMember struct {
 	APILevel uint32
 }
 
-type DaprHostMemberStateData struct {
-	// Index is the index number of raft log.
-	Index uint64
+func (d *DaprHostMember) NameAndNamespace() string {
+	return d.Namespace + "-" + d.Name
```

Comment:
`-` is a valid character in namespaces and names meaning that it is generally not a safe separator to use. Dapr uses `||` instead, and I think we should be consistent even if safety isn't a problem here.
```suggestion
	return d.Namespace + "||" + d.Name
```